### DISCUSSION TITLE
V3 feeds

### DISCUSSION THREAD

**Discussion by JoannSellerServices** [etsy/open-api#939] [Q&A]
in V2 there was a concept of "Feeds" (real time, pubhub, RSS, etc. depicted below in the docs).  V3 doesn't mention any resources like this, but it also doesn't list them in the summary of changes, or the Retired Resources sections.  Does anyone know if any of these features are available in V3?
![image](https://user-images.githubusercontent.com/90358205/230178369-2ae50bc7-ad27-4875-9fa3-2f9650545ab0.png)

**Comment by Gareth064** [comment]
No feeds in V3
